src/Components/Components/src/HotReload/HotReloadManager.cs

```diff
@@ -16,5 +19,7 @@ public static void DeltaApplied()
         {
             OnDeltaApplied?.Invoke();
         }
+
+        public static void OnAfterUpdate(Type[]? _) => OnDeltaApplied?.Invoke();
```

Comment:
For a little while this means it'll be invoking OnDeltaApplied twice for each update?  That's ok?

Reply:
This should be fine. The second render should no-op because none of the parameters would have changed.